src/i18n/it/index.ts

```diff
@@ -1,12 +1,13 @@
 import type { Translation } from '../i18n-types'
 
 const it: Translation = {
+	title: 'typesafe-i18n - Svelte Summit Autunno 2021',
 	HI: 'Benvenuti al Svelte Fall Summit {year}',
```

Comment:
can you please also change the naming for `HI` for `it` and `de`?

Reply:
Done, I took the liberty to rename `HI` to `welcome` since all the other keys are lowercase and "welcome" makes more sense for a welcome message imo.